### PR TITLE
add version specific imagestream tags

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton @ncdc ptal.

**Comment by bparees** [comment]
[test]

File: examples/image-streams/image-streams-centos7.json [hunk 1]
```diff
@@ -184,11 +212,15 @@
         "tags": [
           {
             "name": "latest"
+          },
+          {
+            "name": "9.2",
+            "from": {
+              "Kind": "ImageStreamTag",
+              "Name": "postgresql:latest"
```
Comment (smarterclayton) [inline examples/image-streams/image-streams-centos7.json]: This can just be "latest"

File: examples/image-streams/image-streams-centos7.json [hunk 2]
```diff
@@ -14,18 +14,22 @@
...
+            "name": "2.0",
             "annotations": {
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
               "version": "2.0"
+            },
+            "from": {
+              "Kind": "ImageStreamTag",
+              "Name": "ruby:latest"
```
Comment (ncdc) [inline examples/image-streams/image-streams-centos7.json]: You should just be able to say `latest`. LMK if that doesn't work.

File: examples/image-streams/image-streams-centos7.json (hunk 1 above)
Comment (bparees) [inline examples/image-streams/image-streams-centos7.json]: I was trying to set up the demonstration of having following tags, even if it is currently reversed... do you want me to just make it two independent tags?

File: examples/image-streams/image-streams-centos7.json (hunk 2 above)
Comment (bparees) [inline examples/image-streams/image-streams-centos7.json]: yeah that appears to work, i'll change it.

**Comment by smarterclayton** [comment]
Approved

**Comment by bparees** [comment]
[merge]

**Pull request merged**